Assets/Scripts/EnemyShip.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyShip : MonoBehaviour
+{
+    [SerializeField] private float _playerSpeed = 2f;
+    [SerializeField] private float _inertia = 0.9f;
+
+    [SerializeField] private List<KeyCode> _upArrows;
+    [SerializeField] private List<KeyCode> _downArrows;
+    [SerializeField] private List<KeyCode> _leftArrows;
+    [SerializeField] private List<KeyCode> _rightArrows;
+
+    private float _currentSpeed;
+    private Vector3 _lastMovement;
+    private Vector3 _enemyMousePosition;
+
+    private Vector3 _movement;
```

Comment:
Why is it in the enemy ship class?
The task was to create a ship player 2, which will be an ally of player 1